bot/admin/server/src/main/kotlin/BotAdminService.kt

```diff
@@ -122,7 +119,7 @@ object BotAdminService {
     private val storyDefinitionDAO: StoryDefinitionConfigurationDAO by injector.instance()
     private val featureDAO: FeatureDAO by injector.instance()
     private val dialogFlowDAO: DialogFlowDAO get() = injector.provide()
-    private val front = FrontClient
+    internal val front = FrontClient
```

Comment:
not sure this should be internal. I would prefer you add `private val front = FrontClient` in `BotAdminVerticle`